src/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -246,20 +272,13 @@ private static boolean saveScreenshot(final HistoryNode node, final File file, f
       }
 
       // save Image as .png
-      try {
-        ImageIO.write(mapImage, "png", file);
-      } catch (final Exception e2) {
-        e2.printStackTrace();
-        JOptionPane.showMessageDialog(frame, e2.getMessage(), "Error saving Screenshot", JOptionPane.OK_OPTION);
-        retval = false;
-      }
+      ImageIO.write(mapImage, "png", file);
```

Comment:
The error handling was moved up to the public `saveScreenshot()` overload.  I modified the private overload (which runs on a non-EDT thread, and thus shouldn't display a message dialog) to throw an exception on failure.  That exception is now handled in the public overload (which runs on the EDT), and it is responsible for displaying both the success and failure message dialogs.